src/api/Controllers/UploadController.cs

```diff
@@ -126,21 +126,22 @@ public async Task<ActionResult<int>> UploadFileAsync(int workgroupId, IFormFile
 
     private void ValidateBoreholes(List<Borehole> boreholes)
     {
+        var nullOrEmptyMsg = "Field '{0}' is not provided.";
```

Comment:
REQ: `Field '{0}' is required.` wäre in diesem Fall, wie im Issue vorgeschlagen, besser verständlich. Mit der aktuellen Message könnte man es auch so interpretieren, dass zum Beispiel der entsprechende Header fehlt.